examples/shared_samples/hypervisor_cluster_profile.rb

```diff
@@ -75,5 +79,7 @@
 puts "\nCompliance preview details are :\n#{cp}\n"
 
 # Deletes the created hypervisor cluster profile
-hcp.delete
+# Delete method accepts 2 optional arguments - soft_delete(boolean) and force(boolean)
+# The default values for both the arguments is "false"
+hcp.delete(true, true)
```

Comment:
Please try if hcp.delete(soft_delete = true, force = true)  is working or not. If it is, change it accordingly.